tls.c

```diff
@@ -107,11 +107,19 @@ static bool load_server_certificates(char **errmsg) {
     } else if (!SSL_CTX_check_private_key(settings.ssl_ctx)) {
         snprintf(error_msg, errmax, "Error validating the certificate\r\n");
         success = false;
-    } else {
-        settings.ssl_last_cert_refresh_time = current_time;
+    } else if (settings.ssl_ca_cert) {
+        SSL_CTX_set_client_CA_list(settings.ssl_ctx,
+          SSL_load_client_CA_file(settings.ssl_ca_cert));
+        if (!SSL_CTX_load_verify_locations(settings.ssl_ctx,
```

Comment:
I should have done this first and then do the `SSL_CTX_set_client_CA_list...`. Let's do it like that here.